src/components/CentralHeader/CentralHeader.tsx

```diff
@@ -31,22 +32,42 @@ interface ICentralHeaderProps extends RouteComponentProps {
 /**
  * Central header right below Slim header, containing app title and user name and user role with dropdown to go to profile page and logout
  */
-export const CentralHeader = withRouter<
-  ICentralHeaderProps,
-  React.FC<ICentralHeaderProps>
->((props: ICentralHeaderProps) => {
+export const CentralHeader = withRouter((props: ICentralHeaderProps) => {
   /**
    * react-i18next translation hook
    */
   const { t } = useTranslation();
 
+  /**
+   * add user icon if user name is set
+   */
+  const userIcon =
+    props.userName.trim() !== "" ? (
+      <svg className="icon icon-white">
+        <use xlinkHref={`${bootstrapItaliaImages}#it-user`} />
+      </svg>
+    ) : null;
+
+  /**
+   * switch from user role enum to string
+   */
+  const userRole = (userRoleEnum => {
+    switch (userRoleEnum) {
+      case UserRoleEnum.ORG_DELEGATE:
+        return t("centralHeader.userBadge.delegate");
```

Comment:
I'm a little bit skeptical about the naming of the locales. If these values are supposed to be used somewhere else is better to drop the prefix "centralHeader" and name them with some domain concept (ie. `user.badge.delegate`)

Reply:
I agree with you but I would prefer to make these changes in a separate pr (I created https://www.pivotaltracker.com/story/show/169342054 for this purpose)